components/Image/image.css

```diff
@@ -0,0 +1,16 @@
+.img-container {
```

Comment:
give it the proper of object-fit so that it can have responsiveness

Reply:
ok will do it